src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```diff
@@ -643,6 +643,10 @@
       <Project>{E248B2CA-303B-4645-ADDC-9D4459D550FD}</Project>
       <Name>libZipSharp</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\external\xamarin-android-tools\src\Xamarin.Android.Tools\Xamarin.Android.Tools.csproj">
```

Comment:
We should possibly follow suit with other external modules:

1. In toplevel `Configuration.props`, add a `$(XamarinAndroidToolsFullPath)` MSBuild property (and others)
2. This line should then become:

        <ProjectReference Include="$(XamarinAndroidToolsFullPath)\src\Xamarin.Android.Tools\Xamarin.Android.Tools.csproj">

This is so that the directory can be overridden.